### ISSUE TITLE
A11y_Polyglot Notebooks Extension for VS Code_Table_ Name Role Value: Name is not defined for "Share" buttons.

### ISSUE THREAD

**Issue by kupatkar99** [dotnet/interactive#2463]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Win32; #.NET Interactive; #WCAG4.1.2; #Name Role Value;
#SH-PolyglotNotebooksExtension-Win32-Nov2022; 
### Environment and OS details:
Visual Studio Code - Insiders
Windows 11 Enterprise version 22H2 (OS Build 22623.875)

### Repro Steps:
1)Install "Visual Studio Code" insider application.
2)Install .NET 6.0 SDK using https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.402-windows-x64-installer.
3)Install Polyglot Notebook extension. 
4)launch Visual studio code insider and Ctrl+Alt+Shift+N will invoke "Polyglot Notebook: Create new blank notebook" command used above.
5)Enter any code and run the code.
6)Tab till values and press enter.
7)In Polyglotnotebooks:values section tab till "table". 
8)Verify whether the name is defined for share button or not.

### Actual Result:
Name is not defined for "Share" buttons.

### Expected Result:
Name should be defined for "Share" buttons.
![A11y_ NETInteractive_Table_ AI4D](https://user-images.githubusercontent.com/98530069/201645534-49a8a2a8-1101-417e-aea0-43da3f7c5945.png)

**Comment by VijayaBharathyA** [comment]
#Closed
Verified the issue in Below Environment:
VS code Version: 1.74.0-insider (user setup)
Windows 11 Pro Version 22H2
Windows Feature Experience Pack 1000.22636.1000.0
Issue is fixed. Hence Closing bug
![Name should be defined for Share buttons](https://user-images.githubusercontent.com/100189407/205566326-d380f8bd-7bf5-4a7f-8886-8228ce4042ea.jpg)
